Source/Fuse.Scripting.JavaScript/ThreadWorker.uno

```diff
@@ -83,17 +83,15 @@ namespace Fuse.Scripting.JavaScript
 		{
 			try
 			{
+				_context = Fuse.Scripting.JavaScript.JSContext.Create();
+
 				if (_context == null)
 				{
```

Comment:
While you're at it; I don't think this function can fail, so we shouldn't actually need this if-guard either...